### DISCUSSION TITLE
Can I give height width percentage of any components?

### DISCUSSION THREAD

**Discussion by saurabhwadekar** [flet-dev/flet#1051] [General]
Can I give height width percentage of any components?

**Comment by ndonkoHenri** [comment]
Controls have a `width` property. You could perhaps set it to the percentage of the page's width. But But... I am not sure that calculated percentage will be modified or recalculated automatically by flet.
So you will maybe have to listen to the window's resize events, and force flet to recalculate it.

I haven't tried all this though. Maybe someday.

**Comment by saurabhwadekar** [comment]
Thanks Brother
